sprotty-vscode-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sprotty-vscode",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
We increase the versions when we publish. Please don't change them here.

Reply:
All versions changes are revoked again